DependencyInjection/Configuration.php

```diff
@@ -208,4 +210,56 @@ private function addVisitorsSection(NodeBuilder $builder)
             ->end()
         ;
     }
+
+    private function addContextSection(NodeBuilder $builder)
+    {
+        $builder
+            ->arrayNode('context')
```

Comment:
you should name this differently since we have both, "serialization" and "deserialization" context.

I suggest `default_serialization_context`. What do you think?